docs/README.md

```diff
@@ -20,95 +20,81 @@ Choose either RPM or TAR install.
 
 #### RPM Install
 
-Setup XSEDE RPM repository trust:
-
-  * For production software use [Production Repo Trust Instructions](https://software.xsede.org/production/repo/repoconfig.txt)
-  * For development/testing software use [Development Repo Trust Instructions](https://software.xsede.org/development/repo/repoconfig.txt)
-
-Install package:
-
-     $ yum install access-oauth-mapfile
+Install the desired RPM release from [https://github.com/access-ci-org/access-oauth-mapfile/releases](https://github.com/access-ci-org/access-oauth-mapfile/releases).
 
 The RPM installs files under /usr/local/share/utils/access_oauth_mapfile/ by default.
 
 #### TAR Install
 
-Download the latest Production or Development/Testing tar from either:
-  * [https://software.xsede.org/production/access-oauth-mapfile/latest](https://software.xsede.org/production/access-oauth-mapfile/latest)
-  * [https://software.xsede.org/development/access-oauth-mapfile/latest](https://software.xsede.org/development/access-oauth-mapfile/latest)
+Download the desired TAR release from [https://github.com/access-ci-org/access-oauth-mapfile/releases](https://github.com/access-ci-org/access-oauth-mapfile/releases).
 
 Execute:
 
     $ mkdir {arbitrary_path}/access-oauth-mapfile-<version>
```

Comment:
Actually, the tarball now has access-oauth-mapfile-<version> as the root directory now, so I'll change these instructions

Reply:
I fixed tar install instructions